lib/react.dart

```diff
@@ -1577,6 +1577,27 @@ class SyntheticKeyboardEvent extends SyntheticEvent {
             isTrusted, nativeEvent, target, timeStamp, type) {}
 }
 
+class SyntheticCompositionEvent extends SyntheticEvent {
+  final data;
```

Comment:
Are you okay with making this a string? According to MDN, that's the type: https://developer.mozilla.org/en-US/docs/Web/API/CompositionEvent/data
```suggestion
  final String data;
```

Reply:
Yes good call. Looks like react.js also defines it as a string
https://reactjs.org/docs/events.html#composition-events

Reply:
@greglittlefield-wf fixed